bucket/download.py

```diff
@@ -94,13 +94,17 @@ def askForBucket():
 
 # Download training images.
 image_files = [image for image in annotations.keys()]
+image_files_exceptions = []
 for file in image_files:
     filename = os.path.join(output_dir, file)
     print('saving: {}'.format(file))
     print('to: {}'.format(filename))
     try:
         cos.Object(credentials_1['bucket'], file).download_file(filename)
     except Exception, err:
-        print("Filename: {0}, Client Error: {1}".format(filename, err))
+        image_files_exceptions.append(file)
+        print("File: {0}, Client Error: {1}".format(file, err))
```

Comment:
for consistency can we swap these out for single quotes, and just {} instead of {0}, {1}
I think I would also prefer:
```
print('Error: {}, skipping {}...'.format(err, f))
```